fbxctest/FBXCTestKit/FBXCTestBootstrapper.h

```diff
@@ -0,0 +1,9 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#import <Foundation/Foundation.h>
+
+@interface FBXCTestBootstrapper : NSObject
```

Comment:
We should change the name of this class as it creates ambiguity with the `XCTestBootstrap` Framework. This may become less significant in the future if we decide to merge `XCTestBootstrap` & `FBXCTestKit` into `XCTestControl`